packages/decoupled/src/logger/log-format.ts

```diff
@@ -1,25 +1,47 @@
 import { format } from 'winston';
+import { get } from 'lodash';
+import moment from 'moment';
 import { colorizeLabel } from './colorize-label';
 
 let maxLabelLength = 0;
 
 const updateMaxLabelLength = (label: string) => maxLabelLength = Math.max(maxLabelLength, label.length);
 
 const logFormatter = format.printf((info) => {
+    let label = info.label || '';
+    const timestamp = info.timestamp || '';
+    const level = info.level ? `${info.level} : ` : '';
+    return `${label}${timestamp}${level}${info.message}`;
```

Comment:
@friendken i'd suggest using `array.join(' | ')` instead adding the separator manually to every part